src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/Model/Field.cs

```diff
@@ -22,6 +42,12 @@ public class Field
         [XmlAttribute]
         public string New;
 
+        [XmlAttribute]
+        public string MinCount;
```

Comment:
>MinCount [](start = 22, length = 8)

What are `MinCount` and `AllowTrailingSeparator` for? I didn't see changes related to that.

Reply:
These should have gone in in my two previous PRs, but i didn't realize the change wasn't htere.  basically, i added these two attributes to the nodes, and this is how we can easily use xml deserialization and get access to them.

Reply:
I didn't look at those previous PRs, but I worry that they went in without those properties being declared which indicates that they were not much exercised. Should `MinCount` be validated/enforced by our generated constructors?

Reply:
> they were not much exercised

That is correct, and it was intentional.  The previous PRs (https://github.com/dotnet/roslyn/pull/37680) is intended to be liek this PR.  Namely, it has no runtime impact on roslyn at all.  I did not want to risk anything breaking here.  For example, it's totally reasonable for someone to do this:

`SyntaxFactory.VaraibleDeclaration(type).AddVariables(...)`

Note: i'm not opposed to you takin advantage of this in the future.  However, right now, the 3 changes i've made (mincount/allowtrailing/choice+sequence) are meant to not have any actual impact on roslyn, but to just really make thins nicer for *other* tooling.  

Specifically, i would like roslyn to start generating a g4 grammar like so: https://gist.github.com/CyrusNajmabadi/412c3209d1ce97236420218498e7c8d4  

I'd like this artifact to be produced as part of the build (like all the syntax nodes), and then something that the team, and externals can be pointed to as an easy way to understand some parts of roslyn syntax.  But before contributing that i wanted to improve these three areas to make the grammar better match what people want from it.

Thanks!